Makefile.am

```diff
@@ -35,6 +35,8 @@ endif
 check_LIBRARIES = libcrun_testing.a
 
 libcrun_SOURCES = src/libcrun/utils.c \
+		src/libcrun/blake3/blake3.c \
+		src/libcrun/blake3/blake3_portable.c \
```

Comment:
Would it be a good idea of having a path to sub-repo of blake3 here instead of actual files ? Then crun-repo can avoid maintaining these sources directly ?

Reply:
I made some manual changes to the file: https://github.com/containers/crun/pull/1395/files#diff-99e860d54e0b9d30b4f6ed5f4b08109392e8cb83c0746fdda67f33d50d29f1c7R12-R20

Reply:
Ah sounds good in that case.